action.yml

```diff
@@ -22,7 +22,9 @@ inputs:
     default: ./
 outputs:
   version:
-    description: 'The version'
+    description: 'The value of the new pre-calculated tag'
+  latest-version:
+    description: 'Contains the value of latest tag, before calculating a new one'
```

Comment:
I think it would be more explicit as previous-version
latest-version can be mistaken as the new version